### PR TITLE
Add ACP module component

### LINKED ISSUE #3: ACP Module
An ACP module that allows the admin to enable/disable the sites they want to allow media embedding from.

Also an option to display the media bbcode on posting pages. (Since URLs will be parsed with or without the [media] tag, the bbcode is optional and redundant, but still nice to have for users who are simply unaware that URLs can be embedded in the raw.)

Here's the complete list of supported sites: http://s9etextformatter.readthedocs.io/Plugins/MediaEmbed/Sites/

### PR DESCRIPTION
**Pull request by VSEphpbb**
Closes #3 

Settings:
screen shot 2016-09-29 at 9 41 10 am

Manage sites:
redo

### CODE REVIEW

**Comment by rxu** [comment]
LGTM 👍

**Comment by rxu** [comment]
Again 👍

File: acp/main_module.php [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+	/** @var string $u_action */
+	public $u_action;
+
+	/**
+	 * Constructor
+	 */
+	public function __construct()
+	{
+		global $phpbb_container;
```
Comment (michaelcullum) [inline acp/main_module.php]: *cries a little* Please re-write the ACP module system to use services and tags and I'll love you forever.

Comment (VSEphpbb) [inline acp/main_module.php]: Not possible yet. ACP modules are not aware of the service container.

**Pull request merged**